main-settings/src/main/scala/sbt/Def.scala

```diff
@@ -57,17 +57,36 @@ object Def extends Init[Scope] with TaskMacroExtra {
           ref => displayBuildRelative(currentBuild, multi, ref)
       ))
 
+  /**
+   * Returns a String expression for the given [[Reference]] (BuildRef, [[ProjectRef]], etc)
+   * relative to the current project.
+   */
+  def displayRelativeReference(current: ProjectRef, project: Reference): String =
+    displayRelative(current, true, project, false)
+
+  @deprecated("Use displayRelativeReference", "1.1.0")
   def displayRelative(current: ProjectRef, multi: Boolean, project: Reference): String =
+    displayRelative(current, multi, project, true)
+
+  private[sbt] def displayRelative(current: ProjectRef,
+                                   multi: Boolean,
+                                   project: Reference,
+                                   trailingSlash: Boolean): String = {
+    val slash = if (trailingSlash) "/" else ""
     project match {
-      case BuildRef(current.build)      => "{.}/"
-      case `current`                    => if (multi) current.project + "/" else ""
-      case ProjectRef(current.build, x) => x + "/"
-      case _                            => Reference.display(project) + "/"
+      case BuildRef(current.build) => "ThisBuild" + slash
+      case `current`               =>
+        // ignore (multi) and always display relative key
+        // if (multi) current.project + slash else ""
+        ""
```

Comment:
Could you explain what's going on here? Why is multi ignored now?

Reply:
`multi = true` is passed in whenever you have multi-projects. Prior to this change, when sbt detected a multi-project build it will display `$subproject/key` even though the method is called `displayRelative`. At least the Build URI is omitted, but I am making it *more* relative by omitting the subproject id.

Here's how this would show up. When you run `inspect test` in sbt 1.0.0 you get:

```
[info] Dependencies:
[info] 	sbtRoot/test:executeTests
[info] 	sbtRoot/test:test::streams
[info] 	sbtRoot/test:state
[info] 	sbtRoot/test:test::testResultLogger
```

In sbt 1.1.0 you'd get:

```
[info] Dependencies:
[info] 	Test/executeTests
[info] 	Test/test/streams
[info] 	Test/state
[info] 	Test/test/testResultLogger
```

I think this is better, because what's something the user can type in or use in `build.sbt`.

Reply:
I see. So can we lose the commented out code and the `multi` parameter from this new `private[sbt]` method?